ntoskrnl/ob/obname.c

```diff
@@ -214,26 +214,6 @@ ObpCreateDosDevicesDirectory(VOID)
     if (!NT_SUCCESS(Status))
         goto done;
 
-    /*********************************************\
-    |*** HACK until we support device mappings ***|
-    |*** Add a symlink \??\ <--> \GLOBAL??\    ***|
-    \*********************************************/
-    RtlInitUnicodeString(&LinkName, L"\\??");
-    InitializeObjectAttributes(&ObjectAttributes,
-                               &LinkName,
-                               OBJ_PERMANENT,
-                               NULL,
-                               &DosDevicesSD);
-    Status = NtCreateSymbolicLinkObject(&SymHandle,
-                                        SYMBOLIC_LINK_ALL_ACCESS,
-                                        &ObjectAttributes,
-                                        &RootName);
-    if (NT_SUCCESS(Status)) NtClose(SymHandle);
-    /*********************************************\
-    \*********************************************/
-
-    // FIXME: Create a device mapping for the global \?? directory
```

Comment:
The comment makes it sound like there should be code here?

Reply:
It's done above, actually, the ObSetDeviceMap call.